lib/datadog/ci/ext/settings.rb

```diff
@@ -4,7 +4,7 @@ module CI
     module Ext
       # Defines constants for test tags
       module Settings
-        ENV_MODE_ENABLED = 'DD_TRACE_CI_MODE_ENABLED'.freeze
+        ENV_MODE_ENABLED = 'DD_TRACE_CI_ENABLED'.freeze
```

Comment:
As far as I can tell, this environment variable was introduced here for completeness, not because it's a require for CiApp.

With that in mind, I renamed it to match the new setting name.

This change is documented in the "breaking changes" section of this PR.